src/components/views/rooms/EventTile.tsx

```diff
@@ -1017,7 +1017,7 @@ export class UnwrappedEventTile extends React.Component<IProps, IState> {
 
         return (
             <MessageContextMenu
-                {...aboveLeftOf(this.state.contextMenu.position)}
+                {...aboveRightOf(this.state.contextMenu.position)}
```

Comment:
Could this use an `alwaysBelowRightOf` placement instead? I don't think it's conventional to place context menus above the cursor.

Reply:
This actually puts the menu above/below depending on which half of the screen you click. From what I gather the context-menu positioning system is somewhat flawed and might not allow what we actually want

Reply:
https://user-images.githubusercontent.com/25768714/163706819-228388dc-1f62-45ed-8d23-62710e71d18b.mp4

Does this satisfy your needs or should I try to figure out something better?

Reply:
Hm, right, I suppose that's fine